ideascube/serveradmin/catalog.py

```diff
@@ -604,9 +606,17 @@ def list_nothandled(self, ids):
 
         return sorted(pkgs, key=attrgetter('id'))
 
+    def update_displayed_packages_on_home(self, *, to_remove_ids=[], to_add_ids=[]):
```

Comment:
Only if the arguments are changed in the function. Else it is useless.

And I thing that the `to_remove_ids = to_remove_ids or []` pattern is not so easy to read so I prefer to not use it if not necessary.